### DISCUSSION TITLE
Does this work on the TUF A15, and do you recommend the switch?

### DISCUSSION THREAD

**Discussion by Darena-Bryant** [seerge/g-helper#1478] [Q&A]
Today AC updated on me and tried to introduce new features. I say 'tried' because it changed some of my settings, and when I relogged as it suggested, everything was sorta-kinda back to how it was before the update. Worse, I had lost access to my screen lighting features, and the one feature I had tried to change was mysteriously no longer available (the one that lets you set Ultimate mode for your GPU and such; it was there after the initial update, then gone after the restart). I'm not sure WHAT is going on at this point, but I noticed AC had opened a ton of processes after the restart that it hadn't had before, so shut them all down.

I also tried to connect to 'customer service'. Ha. There's no customer service. After that nightmarish run around, I really would like to remove this program from my system completely. Will my TUF A15 have some kind of spaz out if I do? Is AC required for it to work properly, or can I uninstall it, then install G-Helper in it's place without fear? I'm not very technical or skilled in this kind of thing, and I'm basically homebound, so my laptop is my only source of distraction and entertainment. I'm just trying to be sure it's safe to switch over before doing anything I might not be able to back out of.

**Comment by Kikimahe** [comment]
Hi, 

You can safely uninstall Armory Crate. Your computer will continue to work.
Use the [ASUS clean uninstall Armory tool:](https://dlcdnets.asus.com/pub/ASUS/mb/14Utilities/Armoury_Crate_Uninstall_Tool.zip?model=armoury%20crate)

Then, if you don't like Ghelper, you could reinstall Armory Crate and all those processes and services launched. 
But you will like it. :-)

**Comment by seerge** [comment]
@Darena-Bryant yes, it does work on all modern asus laptops
